test/built-ins/Set/prototype/difference/add-not-called.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2023 Anthony Frehner and Kevin Gibbons. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-set.prototype.difference
+description: Set.prototype.difference should not call Set.prototype.add
+features: [set-methods]
+includes: [compareArray.js]
+---*/
+
+const s1 = new Set([1, 2]);
+const s2 = new Set([2, 3]);
+const expected = [2];
```

Comment:
```suggestion
const expected = [1];
```
right? since 2 is in both sets, "difference" shouldn't include it.

Reply:
Yup, good catch.